src/main/java/io/pravega/perf/PravegaPerfTest.java

```diff
@@ -361,7 +385,7 @@ public static Test createTest(long startTime, CommandLine commandline, Options o
                 } else {
                     action = "Reading";
```

Comment:
Maybe these "actions" could be an enum or a constant something for clarity. In any case, this this variable necessary?

Reply:
The action is really just used as a label when writing out results.  I believe the variable only exists to make the call to the `PerfStats` constructor easier to read (it wasn't introduced in this PR).  They could be moved to a CONSTANT, but since they're only used once here I'm not sure it would anything.

Reply:
These "actions" are just labels used when writing out the results (they weren't added by this PR).  I could change them for constants if you'd prefer, but since they only appear here I'm not sure that really buys anything.

The `action` variable I believe was only introduced in order to make the call to the `PerfStat` constructor cleaner.